docs/source/_static/benchmark.csv

```diff
@@ -0,0 +1,23 @@
+﻿Acronym,TID2013,TID2013,KADID10k,KADID10k
+,SRCC / KRCC (piq),SRCC / KRCC,SRCC / KRCC (piq),SRCC / KRCC 
+PSNR,0.6869 / 0.4958,0.687 / 0.496,0.6757 / 0.4876,- / -
+SSIM,0.5544 / 0.3883,0.637 / 0.464,0.6329 / 0.4571,0.718 / 0.532
+MS-SSIM,0.7983 / 0.5965,0.787 / 0.608,0.8020 / 0.6088,0.802 / 0.609
+VIFp,0.6102 / 0.4579,0.610 / 0.457,0.6500 / 0.4770,0.650 / 0.477
+FSIM,0.8015 / 0.6289,0.801 / 0.630,0.8294 / 0.6390,0.829 / 0.639
+FSIMc,0.8509 / 0.6665,0.851 / 0.667 ,0.8537 / 0.6650,0.854 / 0.665
+IW-PSNR,- / -,0.6913 / - ,- / -,- / -
+IW-SSIM,- / -,0.7779 / 0.5977,- / -,- / -
+SR-SIM,- / -,0.8076 / 0.6406,- / -,0.839 / 0.652
+SR-SIMc,- / -,- / -,- / -,- / -
+GMSD,0.8038 / 0.6334,0.8030 / 0.6352,0.8474 / 0.6640,0.847 / 0.664
+VSI,0.8949 / 0.7159,0.8965 / 0.7183,0.8780 / 0.6899,0.861 / 0.678
+Content,0.7049 / 0.5173,- / -,0.7237 / 0.5326,- / -
+Style,0.5384 / 0.3720,- / -,0.6470 / 0.4646,- / -
+HaarPSI,0.8732 / 0.6923,0.8732 / 0.6923,0.8849 / 0.6995,0.885 / 0.699
+MDSI,0.8899 / 0.7123,0.8899 / 0.7123,0.8853 / 0.7023,0.885 / 0.702
+MS-GMSD,0.8121 / 0.6455,0.8139 / 0.6467,0.8523 / 0.6692,- / -
+MS-GMSDc,0.8875 / 0.7105,0.687 / 0.496,0.8697 / 0.6831,- / -
+LPIPS-VGG,0.6696 / 0.4970,0.670 / 0.497,0.7201 / 0.5313,- / -
+PieAPP,0.8355 / 0.6495,0.875 / 0.710,0.8655 / 0.6758,- / -
+DISTS,0.7077 / 0.5212,0.830 / 0.639,0.8137 / 0.6254,- / -
```

Comment:
Fix the "-" adding "\\" or copy-paste previous table.
![Screenshot 2021-02-14 at 20 00 44](https://user-images.githubusercontent.com/22414094/107883330-6bd88580-6eff-11eb-8379-a9506e79e1c8.png)